Kitodo/src/main/java/de/sub/goobi/forms/MassImportForm.java

```diff
@@ -100,13 +102,19 @@ public MassImportForm() {
     }
 
     /**
-     * Prepare.
+     * Prepare template and project for which new process will be created.
      *
-     * @return String
+     * @param projectId
```

Comment:
You should switch the ordering in JavaDoc to match ordering of method parameters.